### PR TITLE
🌐 [i18n-KO] Translated `troubleshooting.mdx` to Korean

### PR DESCRIPTION
**Pull request by 0525hhgus**
# What does this PR do?

Translated the `troubleshooting.mdx` file of the documentation to Korean.
Thank you in advance for your review 😄 

Part of https://github.com/huggingface/transformers/issues/20179

## Before reviewing
- [x] Check for missing / redundant translations (번역 누락/중복 검사)
- [x] Grammar Check (맞춤법 검사)
- [x] Review or Add new terms to glossary (용어 확인 및 추가)
- [x] Check Inline TOC (e.g. `[[lowercased-header]]`)
- [x] Check live-preview for gotchas (live-preview로 정상작동 확인)

## Who can review? (Initial)

Team PseudoLab, may you please review this PR? 
@0525hhgus, @KIHOON71, @sim-so, @gabrielwithappy, @HanNayeoniee, @wonhyeongseo, @jungnerd

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review? (Final)

May you please review this PR? 
@sgugger, @ArthurZucker, @eunseojo

### CODE REVIEW

File: docs/source/ko/troubleshooting.mdx [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+<Tip>
+
+메모리 절약 기술에 대한 자세한 내용은 성능 [가이드](performance)를 참조하세요.
+
+</Tip>
+
+## 저장된 TensorFlow 모델을 가져올 수 없습니다(Unable to load a saved TensorFlow model)[[unable-to-load-a-saved-uensorFlow-model]]
+
+TensorFlow의 [model.save] 메소드는 아키텍처, 가중치, 훈련 구성 등 전체 모델을 단일 파일에 저장합니다. 그러나 모델 파일을 다시 가져올 때 🤗 Transformers는 모델 파일에 있는 모든 TensorFlow 관련 객체를 가져오지 않을 수 있기 때문에 오류가 발생할 수 있습니다. TensorFlow 모델 저장 및 가져오기 문제를 피하려면 다음을 권장합니다:
```
Comment (0525hhgus) [inline docs/source/ko/troubleshooting.mdx:66]:
```suggestion
TensorFlow의 [model.save](https://www.tensorflow.org/tutorials/keras/save_and_load#save_the_entire_model) 메소드는 아키텍처, 가중치, 훈련 구성 등 전체 모델을 단일 파일에 저장합니다. 그러나 모델 파일을 다시 가져올 때 🤗 Transformers는 모델 파일에 있는 모든 TensorFlow 관련 객체를 가져오지 않을 수 있기 때문에 오류가 발생할 수 있습니다. TensorFlow 모델 저장 및 가져오기 문제를 피하려면 다음을 권장합니다:
```

File: docs/source/ko/troubleshooting.mdx [hunk 2]
```diff
@@ -170,19 +170,19 @@ tensor([[ 0.0082, -0.2307],
         [-0.1008, -0.4061]], grad_fn=<AddmmBackward0>)
 ```
 
-🤗 Transformers doesn't automatically create an `attention_mask` to mask a padding token if it is provided because:
+🤗 Transformers는 패딩 토큰이 제공된 경우 패딩 토큰을 마스킹하기 위해 `attention_mask`를 자동으로 생성하지 않습니다:
```
Comment (sim-so) [inline docs/source/ko/troubleshooting.mdx:173]:
```suggestion
🤗 Transformers는 패딩 토큰이 제공된 경우 패딩 토큰을 마스킹하기 위한 `attention_mask`를 자동으로 생성하지 않습니다. 그 이유는 다음과 같습니다:
```
아래 내용이 위 문장에 대한 이유라는 것을 명시적으로 표현하면 더 좋을 것 같습니다!

**Review by sim-so — CHANGES_REQUESTED** [review]
고생 많으셨습니다! 깔끔하고 멋진 번역 감사합니다👍

File: docs/source/ko/troubleshooting.mdx (hunk 2 above)
Comment (0525hhgus) [reply docs/source/ko/troubleshooting.mdx:173]: 리뷰 감사합니다 😄 더 자연스러운 문장이군요!

**Comment by 0525hhgus** [comment]
May you please review this PR? 😃 
@sgugger, @ArthurZucker, @eunseojo

**Review by wonhyeongseo — APPROVED** [review]
LGTM!

**Comment by sgugger** [comment]
@0525hhgus you need to put it out of draft mode if it's ready for review.

**Comment by 0525hhgus** [comment]
> @0525hhgus you need to put it out of draft mode if it's ready for review.

I changed to ready for review status! Thank you for your review.

**Review by sgugger — APPROVED** [review]
Thanks a lot!
